.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+Use the following template
+
+## FILE 1 TITLE
```

Comment:
- Pull Request template in an "issue template" PR? Might warrant a renaming
- Sorta long for a PR template
- Not a huge fan of the section titles being in caps

Reply:
@ChristopherWMM Yeah it needs to be renamed, I decided to add that to the end of it but forgot to rename.

I only made it long for clarity purposes, so that people would understand the format given multiple files.

Its not intended to be in caps, its just so that people realize to replace them, although I can see the confusion.